src/Expectations.php

```diff
@@ -13,3 +13,14 @@
     // @phpstan-ignore-next-line
     return $this->toBeInstanceOf(\Illuminate\Support\Collection::class);
 });
+
+/*
+ * Asserts the given model exists in the database.
+ */
+expect()->extend('toExist', function (): Expectation {
```

Comment:
Can you remove this? We don't wanna add custom expectations in this plugin.

Reply:
sorry @nunomaduro, maybe I'm missing something

this seems to be ne only way to add an expectation for a laravel environment (like its done for `->toBeCollection()`)

should `expect()->toExist()` be implemented in the core plugin? wouldn't it break for non-laravel environments?

am I wrong?